avocado/plugins/xunit.py

```diff
@@ -71,7 +71,7 @@ def _create_failure_or_error(self, document, test, element_type,
                     if log_size < max_log_size:
                         text_output = logfile_obj.read()
                     else:
-                        size = max_log_size / 2
+                        size = int(max_log_size / 2)
```

Comment:
This works fine, but one other way that avoids the type conversion is to use `//`.

Reply:
oh, true... That would have been better...